firerunner/src/main.rs

```diff
@@ -135,7 +187,7 @@ fn main() {
         drive_id: String::from("rootfs"),
         path_on_host: PathBuf::from(rootfs),
         is_root_device: true,
-        is_read_only: true,
```

Comment:
Using a read only file system for both the rootfs and appfs works for me, and is preferable from a design perspective (we definitely don't want applications changing the shared root file system, and they also shouldn't change their own file system since each boot should start fresh).

Maybe this is a snapshot specific thing?

Reply:
I do not think it is snapshot specific. The error I came across was that kernel paniced when trying to write to /tmp/workload.pid but finding out that the file system is read only.

Reply:
Hrm... Well I don't see why the application needs to do that. My versions of the images don't do that. Besides of we want to write to tmp we should just mount it as a tmpfs (in memory) instead of part of the disk drive.